Index: src/drivers/scsi/qedf/qedf_version.h
===================================================================
--- src.orig/drivers/scsi/qedf/qedf_version.h	2018-08-20 07:17:28.575747708 +0200
+++ src/drivers/scsi/qedf/qedf_version.h	2018-08-20 07:18:58.521888850 +0200
@@ -7,7 +7,7 @@
  *  this source tree.
  */
 
-#define QEDF_VERSION		"8.33.0.20"
+#define QEDF_VERSION		"8.33.0.20_dup7.6"
 #define QEDF_DRIVER_MAJOR_VER		8
 #define QEDF_DRIVER_MINOR_VER		33
 #define QEDF_DRIVER_REV_VER		0
